openedx/adg/lms/applications/views.py

```diff
@@ -213,7 +213,8 @@ def get(self, request):
     def post(self, request):
         """
         Submit user contact information data. If successful, redirects to the experience page.
-        If resume is added, then experience page is skipped and redirect to application cover letter page.
+        If resume is added, then experience page is skipped and redirect to application business line
```

Comment:
*If resume is added, then experience pages is skipped and user is redirected to ...